app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -86,27 +99,27 @@ class InstalledAddonDetailsFragment : Fragment() {
         }
     }
 
-    private fun bindUI(view: View) {
-        val title = addon.translateName(view.context)
+    private fun bindUI(binding: FragmentInstalledAddOnDetailsBinding) {
+        val title = addon.translateName(binding.root.context)
         showToolbar(title)
 
-        bindEnableSwitch(view)
-        bindSettings(view)
-        bindDetails(view)
-        bindPermissions(view)
-        bindAllowInPrivateBrowsingSwitch(view)
-        bindRemoveButton(view)
+        bindEnableSwitch(binding)
+        bindSettings(binding)
+        bindDetails(binding)
+        bindPermissions(binding)
+        bindAllowInPrivateBrowsingSwitch(binding)
+        bindRemoveButton(binding)
```

Comment:
Same as above, mostly wondering if we really need to pass `binding` into each of these bind functions. I can see something like `setAllInteractiveViewsClickable` benefit by just having one less parameter and makes the code much nicer.